### PR TITLE
Android guidelines - Support for changes on the screen

### PR DESCRIPTION
**Pull request by mihaelKampic**
Added guidelines for:
- https://github.com/infinum/accessibility-mobile-standards/issues/44

### CODE REVIEW

File: docs/guidelines/platforms/android/guideline_robust_android.md [hunk 1]
```diff
@@ -138,6 +138,46 @@ in this guideline.
...
+In Compose, the same can be achieved by setting the `liveRegion` semantics property.
+
+Example: Add assertive announcement of changes to the TextField
+
+```
+TextField(
+    modifier = Modifier.semantics {
+        liveRegion = LiveRegionMode.Assertive
+    }
+)
```
Comment (zavrskina) [inline docs/guidelines/platforms/android/guideline_robust_android.md:172]: I would suggest we also add some specific use cases together with code examples. For example:
When the time of cooking changes, the value of the time text field is update and the update is announced... (this is only example that I could think of that might fit to this particular guideline).

**Review by zavrskina — COMMENTED** [review]
🎖️

File: docs/guidelines/platforms/android/guideline_robust_android.md (hunk 1 above)
Comment (mihaelKampic) [reply docs/guidelines/platforms/android/guideline_robust_android.md:176]: @zavrskina updated

File: docs/guidelines/platforms/android/guideline_robust_android.md [hunk 2]
```diff
@@ -138,6 +138,50 @@ in this guideline.
...
+This means that the user should be able to know the status of the app at any given time without having to interact with it.
+
+On mobile this means that the user should be able to know if the current state has been changed - which is especially important for users who rely on screen readers.
+
+> This guideline covers point *4.1.3 Status messages - Level AA of the WCAG standard.*
+
+#### ✅ Success technique(s)
+
+When an important thing on the screen changes, users should be notified about it.
+This can be achieved in View system by by setting `ViewCompat.setAccessibilityLiveRegion` which makes the accessibility service announce
```
Comment (Aksi0m) [inline docs/guidelines/platforms/android/guideline_robust_android.md:153]: I like the by by (👋 ) setting

**Review by Aksi0m — COMMENTED** [review]
Cool thing, I never knew this existed in the Android framework 👍

**Review by Aksi0m — APPROVED** [review]

**Review by zavrskina — APPROVED** [review]
Looks good, I really like the added examples! 🙌

**Pull request merged**